src/CodeFlowDecorations.ts

```diff
@@ -188,60 +233,90 @@ export class CodeFlowDecorations {
      * @param idText text version of the id of the Code Flow, set to undefined if using id
      * @param idCFStep Id object of the Code Flow, set to undefined if using idText
      */
-    public static updateCodeFlowSelection(idText?: string, idCFStep?: CodeFlowStepId) {
-        let id: CodeFlowStepId;
-        if (idText !== undefined) {
+    public static async updateCodeFlowSelection(explorerController: ExplorerController, idText?: string, idCFStep?: CodeFlowStepId): Promise<void> {
```

Comment:
> public static async updateCodeFlowSelection( [](start = 3, length = 45)

These static methods do not need to exist (here and below).

Make the contract between the "Explorer controller" and everything else more clear.